src/plugins/elasticsearch/lib/version_satisfies.js

```diff
@@ -0,0 +1,16 @@
+var semver = require('semver');
+
+module.exports = function (actual, expected) {
+  try {
+    var ver = cleanVersion(actual);
+    return semver.satisfies(ver, expected);
+  } catch (err) {
+    return false;
+  }
+
+  function cleanVersion(version) {
+    var match = version.match(/\d\.\d\.\d/);
```

Comment:
It might not be an issue for awhile, but should a `+` be added after each `\d`, for multi-digit numerics?